test/new-e2e/pkg/components/remotehost.go

```diff
@@ -43,21 +44,33 @@ var _ e2e.Initializable = &RemoteHost{}
 // Init is called by e2e test Suite after the component is provisioned.
 func (h *RemoteHost) Init(ctx e2e.Context) error {
 	h.context = ctx
-	return h.ReconnectSSH()
+	return h.reconnectSSH()
 }
 
 // Execute executes a command and returns an error if any.
 func (h *RemoteHost) Execute(command string, options ...ExecuteOption) (string, error) {
+	var err error
+	var output string
+
 	params, err := optional.MakeParams(options...)
 	if err != nil {
 		return "", err
 	}
+
 	cmd := h.buildEnvVariables(command, params.EnvVariables)
+	output, err = clients.ExecuteCommand(h.client, cmd)
 
-	output, err := clients.ExecuteCommand(h.client, cmd)
+	if err != nil && strings.Contains(err.Error(), "failed to create session:") {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Have you considered checking the error type vs the message ?

Reply:
I tried, but the error returned is not always an `OpenChannelError` ([here](https://github.com/golang/crypto/blob/master/ssh/mux.go#L328-L336)) so it was not really working